### ISSUE TITLE
Error in depicting differential genesets while using  irGSEA.bubble,  irGSEA.bubble,irGSEA.barplot

### ISSUE THREAD

**Issue by sukyorsuki** [chuiqin/irGSEA#9]
irGSEA.heatmap.plot <- irGSEA.heatmap(object = result.dge,
+                                       method = "ssgsea",
+                                       top = 50,
+                                       show.geneset = NULL)
Error in `chr_as_locations()`:
! Can't rename columns that don't exist.
x Column `p_val_adj` doesn't exist.
Run `rlang::last_error()` to see where the error occurred.

irGSEA.bubble.plot <- irGSEA.bubble(object = result.dge,
+                                     method = "ssgsea",
+                                     show.geneset = NULL,
+                                     top = 50)
Error in `chr_as_locations()`:
! Can't rename columns that don't exist.
x Column `p_val_adj` doesn't exist.
Run `rlang::last_error()` to see where the error occurred.

ps, My 'result.dge' contains the  `p_val_adj`

How to fix this error? THX

**Comment by chuiqin** [comment]
是的，这是一个BUG。这个BUG发生，是因为在执行irGSEA.score函数的时候，没有全选这4个方法("AUCell", "UCell", "singscore", "ssgsea")。然后，在执行irGSEA.bubble/irGSEA.bubble/irGSEA.barplot函数的时候，语句发生了错误
```object[1:4] <- object[1:4] %>% purrr::map( ~.x %>% dplyr::rename(pvalue = p_val_adj))```。我在新版本的irGSEA里面修复了这个BUG。在新版本的irGSEA中，即使在执行irGSEA.score函数的时候，我们没有全选这4个方法，也可以正常出图。

**Comment by chuiqin** [comment]
Yes, this is a bug. This bug occurs because the 4 methods ("AUCell", "UCell", "singscore", "ssgsea") are not selected when executing the irGSEA.score function. Then, when executing the irGSEA.bubble/irGSEA.bubble/irGSEA.barplot function, the statement has an error
````object[1:4] <- object[1:4] %>% purrr::map( ~.x %>% dplyr::rename(pvalue = p_val_adj))```. I fixed this bug in the new version of irGSEA. In the new version of irGSEA, even when we execute the irGSEA.score function, we can still draw pictures normally without selecting all the 4 methods.
